### PR TITLE
Pad encoder outputs in attention scoring

### PR DESCRIPTION
**Pull request by hirofumi0810**
- Pad encoder outputs with -1024.0
- Add scaling=1.0 option to multi-head score functions
- Fix scheduled sampling (remove F.log_softmax)

### CODE REVIEW

**Comment by sw005320** [comment]
Can you commit only padding part? Also please report the ASR and ST results with and without this change.
